ArduPlane/mode_rtl.cpp

```diff
@@ -18,6 +18,15 @@ bool ModeRTL::_enter()
             return true;
         }
 
+        // treat RTL as QLAND if we are armed and idling in a q-mode on the ground
+        // (when disarmed, allow RTL mode, so pilots can test switches)
+        if (plane.arming.is_armed() &&
+                plane.control_mode->is_vtol_mode() &&
+                plane.quadplane.motors->get_desired_spool_state() == AP_Motors::DesiredSpoolState::GROUND_IDLE) {
```

Comment:
It would be nice to have a `!plane.is_flying()` in here too if it would still work in your use case. I wonder what Qland actually does in this case? One would hope it would just disarm, if it does I wonder if we would be better to just disarm in RTL and not change mode.